gril/grilreply.c

```diff
@@ -1456,3 +1461,99 @@ struct parcel_str_array *g_ril_reply_oem_hook_strings(GRil *gril,
 out:
 	return str_arr;
 }
+
+struct reply_radio_capability *g_ril_reply_parse_get_radio_capability(
```

Comment:
In the new version I have used this function in two places: radio-settings driver and ril plugin, so it is useful to have this in grilreply. Let's leave this here for the moment and refactor everything when we update to ofono 1.18 (which has been recently released).